website/static/js/koHelpers.js

```diff
@@ -195,6 +196,12 @@ ko.validation.rules.mustEqual = {
     message: 'The field does not match the required input.'
 };
 
+ko.validation.rules.complexity = {
```

Comment:
Is this validation rule used anywhere outside of `setPassword.js`? If not, it might be better to set this there so that `zxcvbn` isn't `require`'d where it isn't needed.